src/org/daisy/dotify/api/formatter/TableOfContents.java

```diff
@@ -21,4 +21,23 @@
 	 */
 	public void endEntry();
 	
+    /**
+	 * Starts a new entry-on-resumed with the supplied properties.
+	 * 
+	 * @param range the range of this entry on resumed
+	 */
+	public void startEntryOnResumed(String range);
+	
+	/**
+	 * Ends the current entry-on-resumed.
+	 */
+	public void endEntryOnResumed();
```

Comment:
Have you thought about reusing `endEntry`?

Reply:
Done.